snap/snapcraft.yaml

```
@@ -69,14 +69,16 @@ parts:
   gegl:
     after:
       - babl
-    source: https://gitlab.gnome.org/GNOME/gegl.git
+    source: https://github.com/GNOME/gegl.git
     source-tag: GEGL_0_4_50
     plugin: meson
     meson-parameters:
       - --prefix=/usr
       - --buildtype=release
       - -Ddocs=false
       - -Dworkshop=true
+    build-environment:
+      - C_INCLUDE_PATH: /snap/gnome-46-2404-sdk/current/usr/lib/x86_64-linux-gnu/pkgconfig/../../../include/librsvg-2.0:${C_INCLUDE_PATH:+:$C_INCLUDE_PATH}
```

```suggestion
      - C_INCLUDE_PATH: /snap/gnome-46-2404-sdk/current/usr/include/librsvg-2.0:${C_INCLUDE_PATH:+:$C_INCLUDE_PATH}
```

Is this not equivalent?

---

Yep, I was just preserving the form of the path used in `librsvg`'s pkgconfig file, but not a big deal for me. I just committed your suggested change.